src/og/webgl/Handler.js

```diff
@@ -1043,7 +1043,7 @@ class Handler {
         }
         gl.deleteBuffer(tmp);
 
-        let numTextureUnits = gl.getParameter(gl.MAX_TEXTlURE_IMAGE_UNITS);
+        let numTextureUnits = gl.getParameter(gl.MAX_TEXTURE_IMAGE_UNITS);
```

Comment:
Interesting, never heard about this method...could you provide a source that it is a proper method to deal with textures?

Reply:
Sorry, I do not remember how I found this; I think it's when I reviewed the code to find a way to add a background colour to labels. Unfortunately I will not be able to provide you with a piece of code.